src/cryo_et_sample_data/_hiv.py

```diff
@@ -20,3 +22,24 @@
 }
 
 hiv = DataSet.from_dict(hiv_config)
+
+
+def _hiv_sample_tomogram() -> List["LayerDataTuple"]:  # noqa: F821
```

Comment:
@alisterburt , do you know how to make flake8 accept the forward reference? I'd rather not depend on napari just for this type annotation. This `noqa` works - just wondering if there's a better way.

Reply:
I *think* this should work at the top of the file

```python
from __future__ import annotations
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from napari.types import LayerDataTuple
```